packages/mapviewer/src/modules/menu/components/advancedTools/MenuAdvancedToolsList.vue

```diff
@@ -34,10 +42,12 @@ function onToggleCompareSlider() {
             ...dispatcher,
         })
     }
-    store.dispatch('setCompareSliderActive', {
-        compareSliderActive: !isCompareSliderActive.value,
-        ...dispatcher,
-    })
+    if (!hasNoActiveLayersWarning.value) {
+        store.dispatch('setCompareSliderActive', {
+            compareSliderActive: !isCompareSliderActive.value,
+            ...dispatcher,
+        })
+    }
```

Comment:
I would do it like this : 
``` js
store.dispatch('setCompareSliderActive', {
        ...dispatcher,
            compareSliderActive: !hasNoVisibleLayer && !isCompareSliderActive.value,
    })
            ...dispatcher,
        })
```

this has the advantage of deactivating the compare slider wen clicking with no layers, should it be active for some reason.